### ISSUE TITLE
 Could not find artifact com.p ingidentity.samples:spring-boot-sdk:jar:0.0.6-SNAPSHOT in github (https://raw.github.com/pingidentity/pingone-customers-spring-boot-tools/mvn-repo/) 

### ISSUE THREAD

**Issue by nehamallama** [pingidentity/pingone-customers-sample-registration#9]
Could not find artifact com.pingidentity.samples:spring-boot-sdk:jar:0.0.6-SNAPSHOT in github (https://raw.github.com/pingidentity/pingone-customers-spring-boot-tools/mvn-repo/) 

Ive tried this on both mac and windows. Using the token key that has been set up in the settings.xml file

**Comment by nehamallama** [comment]
who ever created these docs should be deleted. horrible job

**Comment by jason-hatchett** [comment]
It looks like this dependency was changed about a month ago and something got crossed in the wires. That snapshot is on the master branch of a different project, and this is looking at the wrong branch. I believe a change is needed here: https://github.com/pingidentity/pingone-customers-sample-registration/blob/1eb556f36d98a56cee88330e04db36ded6c5b74d/spring-boot/pom.xml#L57 to move to master.

**Comment by nehamallama** [comment]
looks like line 57 in tree 1eb556f is the same as master already? what needs to be changed in the pom to get this thing to build properly? if you look in the pingone-customers-spring-boot-tools repo that were referencing, it looks like only com.pingidentity.samples:spring-boot-sdk:jar:0.0.1-0.0.4-SNAPSHOT are the available versions.

**Comment by Deivur** [comment]
@nehamallama thank you for finding. You don't need any changes. I have added a new artifact to the repository. It is now available for download as a dependency through a pom file.
`<dependency>
      <groupId>com.pingidentity.samples</groupId>
      <artifactId>spring-boot-sdk</artifactId>
      <version>${project.version}</version>
    </dependency>`

**Comment by Deivur** [comment]
@nehamallama could you please verify?

**Comment by nehamallama** [comment]
The OAuth2SecurityConfigurerAdapter.java file errors out because of the improper annotations @Configuration and @Order(1) at line 20/22, I attached the error trace via the image below. If you comment out these two lines it works but obviously carry out this last change so things will build/run properly directly from the repo. Closing for you now. Thanks so much for the prompt, strong, response. ~Best
![image](https://user-images.githubusercontent.com/25394463/78153795-b1c85000-73f8-11ea-8427-ce470066dc7c.png)
